desktop/apps/consign/components/app/index.js

```diff
@@ -0,0 +1,97 @@
+import React from 'react'
+import ResponsiveWindow from 'desktop/components/react/responsive_window'
+import SubmissionFlow from 'desktop/apps/consign/components/submission_flow'
+
+import { Redirect, Route, Switch } from 'react-router'
+import { data as sd } from 'sharify'
```

Comment:
Due to the last commit [my comment](https://github.com/artsy/force/pull/1722#discussion_r134067059) got lost, so I’m reposting it here:

> Is this supposed to run on the server or client only? Because, if so, the below `sd.CURRENT_USER` access is not safe.